src/playwright-api/makeTest.ts

```diff
@@ -21,9 +22,31 @@ export const makeTest = (
     PlaywrightWorkerArgs & PlaywrightWorkerOptions
   >
 ) =>
-  base.extend<{ save: void }>({
+  base.extend<ChromaticConfig & { save: void }>({
+    // ChromaticConfig defaults
+    delay: [undefined, { option: true }],
+    diffIncludeAntiAliasing: [undefined, { option: true }],
+    diffThreshold: [undefined, { option: true }],
+    disableAutoCapture: [false, { option: true }],
+    forcedColors: [undefined, { option: true }],
+    pauseAnimationAtEnd: [undefined, { option: true }],
+    prefersReducedMotion: [undefined, { option: true }],
+
```

Comment:
So, this is pretty nitpicky, but would it be possible to add single line comments to each of these describing, briefly, what they control?